src/_document/getDocumentFile.ts

```diff
@@ -26,6 +25,6 @@ export default function getDocumentFile({
 
 function getDefaultDocumentFile(): PageFile<NextDocumentFile> {
   return loadFile<NextDocumentFile>({
-    absolutePath: path.resolve(__dirname, 'DefaultDocument.tsx'),
+    absolutePath: 'next/document',
```

Comment:
Do we intentionally import it like this to resolve it in "server" environment?

Reply:
Oui, maybe It's overkill?

Reply:
Could be, but perhaps it's a safer option.